proxy/src/outbound.rs

```diff
@@ -54,6 +54,60 @@ impl<B> Outbound<B> {
             bind_timeout,
         }
     }
+
+    const DEFAULT_PORT: Option<u16> = Some(80);
+
+    /// TODO: Return error when `HostAndPort::normalize()` fails.
+    /// TODO: Use scheme-appropriate-default
+    fn normalize(authority: &http::uri::Authority) -> Option<HostAndPort> {
+        HostAndPort::normalize(authority, Self::DEFAULT_PORT).ok()
+    }
+
+    /// Determines the logical host:port of the request.
```

Comment:
NIT: I think the preferred convention for rustdoc comments is for the summary of a function to be a present-tense description of the action that the function does, like "Determine the logical host:port..." rather than "Determines..."

Reply:
(also I think the bikeshed should be blue...)

Reply:
@hawkw From everything I've seen, the idiomatic style is to conjugate them as I did here.

https://doc.rust-lang.org/src/core/convert.rs.html#309
https://doc.rust-lang.org/src/alloc/vec.rs.html#1091
etc

Reply:
@olix0r Yeah, it looks like [TRPL also uses that conjugation](https://doc.rust-lang.org/book/second-edition/ch14-02-publishing-to-crates-io.html#making-useful-documentation-comments), color me wrong then!

I'm not sure where that got into my head from...